packages/eas-cli/src/build/ios/prepareJob.ts

```diff
@@ -45,9 +46,16 @@ export async function prepareJobAsync(
     }
   }
 
-  const maybeCustomBuildConfigPath = buildProfile.config
```

Comment:
If the logic here is the same for ios and android maybe in `getCustomBuildConfigPath`?

Reply:
This crossed my mind too, I figured we may want to keep `getCustomBuildConfigPath` locale to current environment, e.g. for `eas build --local`. I guess `prepareJob` is the borderline between "we're running on this computer" and "sending this to the server".

Reply:
Yeah, you are right, we should keep this logic in a single place. Since `getCustomBuildConfigPath` can be also used in other context than sending the data to the EAS server I created a new function just for preparing this data for the Job object 👍